setup.py

```diff
@@ -56,7 +56,7 @@ def _get_plugin_version() -> str:
     install_requires=[
         "dbt-spark>=1.8.0, <2.0",
         "dbt-core~=1.8.0",
-        "dbt-adapters~=1.2.0",
+        "dbt-adapters>=1.2.0, <2.0",
```

Comment:
```suggestion
        "dbt-adapters>=1.3.0, <2.0",
```

for `get_catalog_for_single_relation` from 1.3.0 ([link](https://github.com/dbt-labs/dbt-adapters/blob/b64d3a5efd83b8a3ab0612145f6bdeaa8cb0f2ba/.changes/1.3.0.md#L3))